cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorManualRollingUpdatesTest.java

```diff
@@ -429,6 +430,157 @@ vertx, new PlatformFeaturesAvailability(false, KUBERNETES_VERSION),
                 })));
     }
 
+    @Test
+    public void testManualPodRollingUpdateWithPodSetsWithError1(VertxTestContext context) {
```

Comment:
These are effectively parametrized tests, so it might be worth using the actual annotation, it would save having lots of tests with the same name but different numbers at the end. https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-MethodSource